content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/usage.rst

```diff
@@ -1,64 +1,10 @@
-==========================================
-Units of measure, packages, and packagings
-==========================================
+========================
+Packages, and packagings
+========================
```

Comment:
```suggestion
=======================
Packages and packagings
=======================
```

Reply:
KC is correct here. Even if this doc is going to get changed in another PR we can't leave an incorrect doc title like this.